lib/omniship/newgistics.rb

```diff
@@ -0,0 +1,29 @@
+require 'omniship/newgistics/track'
+
+module Omniship
+  module Newgistics
+    LABEL = "Newgistics"
+    TRACKING_REGEX = /TODO/
```

Comment:
thats the barcode but in testing I'm actually tracking it with a "reference number": 123456789A

we'll probably have to wait on some real data from them to figure this out...?